front_end/src/app/(main)/accounts/profile/[id]/page.tsx

```diff
@@ -72,18 +77,24 @@ export default async function Profile({ params: { id }, searchParams }: Props) {
             </span>
           )}
           {currentUser?.is_staff && (
-            <div className="mt-2 flex gap-3 text-sm">
-              {!profile.is_spam && <SoftDeleteButton id={id} />}
-              <span
-                className={`rounded px-2 py-1 ${profile.is_active ? "bg-green-100 text-green-800" : "bg-red-100 text-red-800"} dark:bg-opacity-20`}
-              >
-                {profile.is_active ? "Active" : "Inactive"}
-              </span>
-              <span
-                className={`rounded px-2 py-1 ${profile.is_spam ? "bg-red-100 text-red-800" : "bg-green-100 text-green-800"} dark:bg-opacity-20`}
-              >
-                {profile.is_spam ? "Spam" : "Not Spam"}
-              </span>
+            <div className="mt-2 flex flex-col gap-3 text-sm md:flex-row">
+              <div className="flex flex-wrap items-center gap-3">
+                <Button
+                  href={`/admin/users/user/${profile.id}/change/`}
+                  target="_blank"
+                >
+                  {t("viewInDjangoAdmin")}
+                </Button>
+                {!profile.is_spam && <SoftDeleteButton id={id} />}
+              </div>
+              <div className="flex flex-wrap items-center gap-3">
+                <ProfileChip variant={profile.is_active ? "success" : "danger"}>
+                  {profile.is_active ? "Active" : "Inactive"}
+                </ProfileChip>
+                <ProfileChip variant={profile.is_spam ? "danger" : "success"}>
+                  {profile.is_spam ? "Spam" : "Not Spam"}
+                </ProfileChip>
+              </div>
```

Comment:
Awkwardly, there is a functional difference between "is_staff" and "is_superuser" where "is_staff" means you have permission to soft delete users etc, but "is_superuser" gives the user the ability to access the admin panel.

It's not actually important, but the "viewInDjangoAdmin" button should check if `currentUser?.is_superuser` instead of `is_staff`.